src/modules/getBullsAndCows.js

```diff
@@ -13,7 +13,23 @@
  * Example: { bulls: 1, cows: 2 }
  */
 function getBullsAndCows(userInput, numberToGuess) {
-  /* Write your code here */
+  const result = {
+    bulls: 0,
+    cows: 0,
+  };
+
+  const userInputToString = userInput.toString().split('');
+  const numberToGuessToString = numberToGuess.toString().split('');
+
+  for (let i = 0; i < userInputToString.length; i++) {
+    if (userInputToString[i] === numberToGuessToString[i]) {
+      result.bulls++;
+    } else if (numberToGuessToString.includes(userInputToString[i])) {
+      result.cows++;
+    }
+  }
```

Comment:
The current implementation of the loop is correct, but you might consider using a more functional approach with Array methods like 'forEach' for better readability. However, this is more of a stylistic choice and the current loop is perfectly acceptable if you prefer this style.